### PR TITLE
Release v0.16.0

### PR DESCRIPTION
**Pull request by ni4**
As title says.

### CODE REVIEW

**Comment by ni4** [comment]
@ronaldtse @ribose-jeffreylau @antonsviridenko Could you please review updates to the changelog? Also LGTM/Codecov checks fail but I think we may ignore those, as we were checking things once pushing to the master.

**Review by ribose-jeffreylau — APPROVED** [review]

**Comment by ni4** [comment]
@antonsviridenko Any suggestions? Or time to merge and release :) ?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,52 @@
 ## Changelog
 
+### 0.16.0 [2022-01-20]
+
+#### General
+
+* Added support for OpenSSL cryptography backend so RNP may be built and used on systems without the Botan installation.
```
Comment (antonsviridenko) [inline CHANGELOG.md:7]: maybe "without Botan installed" ?

**Review by antonsviridenko — APPROVED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (ni4) [reply CHANGELOG.md:7]: Ok, fixing! Any other suggestions (to not force-push after the every change).

Comment (antonsviridenko) [reply CHANGELOG.md:7]: no, looks good

Comment (ni4) [reply CHANGELOG.md]: Thanks, re-pushed!

**Comment by ni4** [comment]
Thanks all! Monday is a good day for the release, let's do it.

**Pull request merged**